src/sentry/static/sentry/app/components/events/eventEntries.tsx

```diff
@@ -103,14 +126,19 @@ class EventEntries extends React.Component {
     );
   }
 
-  recordIssueError(errorTypes, errorMessages) {
+  recordIssueError(errorTypes: any[], errorMessages: any[]) {
     const {organization, project, event} = this.props;
+
+    if (!organization) {
```

Comment:
according to the types above `organization` is required, otherwise the component would crash... so I believe you can remove this condition